src/__swaps__/screens/Swap/components/SearchInput.tsx

```diff
@@ -180,7 +183,19 @@ export const SearchInput = ({
         <Column width="content">
           <Animated.View style={buttonVisibilityStyle}>
             <GestureHandlerV1Button
-              onPressJS={() => (output ? outputSearchRef : inputSearchRef).current?.blur()}
+              onPressJS={() => {
+                (output ? outputSearchRef : inputSearchRef).current?.blur();
+
+                console.log(outputProgress.value);
+                if (output && outputProgress.value === NavigationSteps.TOKEN_LIST_FOCUSED && !internalSelectedOutputAsset.value) {
+                  // its the output field, not focused and no selected asset, means button is in "paste" state
+                  Clipboard.getString().then(text => {
+                    if (text.length < 10 || isAddress(text)) {
+                      useSwapsStore.setState({ outputSearchQuery: text });
```

Comment:
this refreshes the list correctly but doesn't update the field value, help

Reply:
You can do it like this:

https://github.com/rainbow-me/rainbow/blob/4e6950ff7d89d48d24d661a5b3c7ecb550e9fa9b/src/components/DappBrowser/search-input/SearchInput.tsx#L295-L302

Probably easiest to use a shared and set it from `onPressJS`

The `text` prop in `useAnimatedProps` is where you want to insert it:

https://github.com/rainbow-me/rainbow/pull/5804/files/1f7912db4f3308093ea879e41c444ac4e23509e3#diff-c9a3f3e749da6c87c415b6fe30db4d980e1791b60f5c9000feae3ebe9b6eda9bR98-R106